yarn.lock

```diff
@@ -2780,6 +2780,17 @@ italia-utils@^4.0.1, italia-utils@^4.1.0:
     swagger-parser "^7.0.0"
     yargs "^11.1.0"
 
+italia-utils@aymen94/io-utils-build:
+  version "4.1.0"
+  resolved "https://codeload.github.com/aymen94/io-utils-build/tar.gz/4a59a9bc23301bd30d37542dccb3faf8394f618f"
```

Comment:
Why does it come from aymen94 and not from a teamdigitale repo?

Reply:
The gen-api-models tool of the io-utils library from the teamdigitale repo doesn't support openapi format but only swagger format. There is [this PR](https://github.com/teamdigitale/io-utils/pull/118) from teamdigitale repo which aims to add backward-compatible OAS3 support, but even if the code has been tested, it still has to be reviewed and this activity has not been scheduled. Moreover, that branch only contains the typescript code, which needs to be transpiled in order to be integrated and used in a project, so it's not possible to directly import it. Hence we decided to import the io-utils library from https://github.com/aymen94/io-utils-build, which contains the transpiled code from [teamdigitale branch](https://github.com/teamdigitale/io-utils/tree/support_openapi3).
This choice has been made on @ioggstream's advice until the PR above is merged, then we can change the import and use the version from teamdigitale repo. Maybe I should add this info to the README file?

Reply:
@cloudify @Undermaken is this something we should wait for, or should we just move forward and merge the patch (and update/fix this in the future)?

Reply:
@LucaPrete reviewing/merging that PR is currently low priority, until when we properly review it, test it and merge it it has to be considered unstable and unsupported, so if there's bugs, there's nobody to fix them - since we're in hurry with little time and resources, my suggestion is to go for the current swagger specs that are well tested and supported across all IO's projects - your call (cc @mccalv )

Reply:
As agreed for the moment we proceed pointing to the branch and, with low priority, we will try to review and merge the pull request. The regression risk is quite low, and this is a good opportunity to move on to OpenAPI 3.